tools/bin/ci_check_dependency.py

```diff
@@ -32,7 +32,10 @@ def main():
 
     # Try to find dependency in build.gradle file
     depended_connectors = list(set(get_depended_connectors(changed_modules, all_build_gradle_files)))
-    write_report(depended_connectors)
+
+    # Create comment body to post on pull request
+    if depended_connectors:
```

Comment:
A question due to me not knowing python that well - is `if 0` false-y?  That's how JS does it

Reply:
Yep, false-y, but not false (`depended_connectors == False` evaluates to False)
```
>>> if [2]:
...     print("test")
...
test
>>> if []:
...     print("test")
...
>>> [] == False
False
```

`if` on an empty list will evaluate to False - as far as I know it's a common python pattern. If you want to check whether the list object doesn't exist at all, you have to `if depended_connectors is not None` - in that case an empty list still counts as something